projects/tests.py

```diff
@@ -97,132 +99,32 @@ def setUpTestData(cls):
             catalog_version="NIST 800-53",
             impact_level="low",
             location="other",
-            creator=cls.test_user,
-            catalog=cls.test_catalog,
-        )
-
-    # Tests for field labels
```

Comment:
These were removed for being too fine-grained. At this point it's just testing django, which is unnecessary. That being said, if folks spot one that shouldn't have been removed that I removed by accident, let me know.